server/migrations/20190820093717-create-question.js

```diff
@@ -9,13 +9,22 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       description: {
-        type: Sequelize.STRING
+        unique: 'compositeIndex',
+        allowNull: false,
+        type: Sequelize.Text
       },
-      subjectId: {
-        type: Sequelize.INTEGER
+      courseId: {
+        unique: 'compositeIndex',
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        onDelete: 'CASCADE',
+        references: { model: 'Courses', key: 'id', as: 'courseId', }
       },
       optionId: {
-        type: Sequelize.INTEGER
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        onDelete: 'CASCADE',
```

Comment:
You cannot make this CASCADE, we cannot delete a question just because we remove the answer, I think you should leave it as **NO ACTION** which is the default